config/routes.rb

```diff
@@ -2,7 +2,12 @@
 
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  # resource :users
+  root 'users#index'
+  resources :users, only: [index, :show] do
+    resources :posts, only: [index, :show] do
+  end
+  # get "/users/:id/posts/:id", to "posts#show"
   # Defines the root path route ("/")
   # root "articles#index"
 end
```

Comment:
- Hi you did a nice job but there are some syntax errors coming from your routers.rb when running your app
- [ ] The First one is you miss an end-of-input `end`.  you closed the input opened on line 7(if I follow the indentation) but you also have to close the one on line 8.
- [ ] The second one is you have to replace your `[index, :show]` with `[:index, :show]` on line 7 and 8